### PR TITLE
[execution] tweak benchmark

### PR DESCRIPTION
**Pull request by zekun000**
## Motivation

Separate txn generation with execution. Add txn_benches to vm benchmarks.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

cargo run --release -p executor-benchmark -- --num-accounts 1000 --block-size 1000 --num-transfer-blocks 10

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

File: execution/executor-benchmark/src/lib.rs [hunk 1]
```diff
@@ -35,7 +35,6 @@ use std::{
     path::PathBuf,
     sync::{mpsc, Arc},
 };
-use storage_client::StorageClient;
```
Comment (lightmark) [inline execution/executor-benchmark/src/lib.rs:38]: why?

Comment (zekun000) [reply execution/executor-benchmark/src/lib.rs:38]: it's too slow, 30% performance drop using this client

File: execution/executor-benchmark/src/lib.rs [hunk 2]
```diff
@@ -70,14 +69,14 @@ struct TransactionGenerator {
 
     /// Each generated block of transactions are sent to this channel. Using `SyncSender` to make
     /// sure if execution is slow to consume the transactions, we do not run out of memory.
```
Comment (wqfish) [inline execution/executor-benchmark/src/lib.rs:71]: Is this comment no longer valid?

**Comment by msmouse** [comment]
Did it really speed up the benchmark by removing the txn generating thread while executor is doing its work? Does it help if we run the benchmark on a better machine? This limits the scale at which we run the benchmark.

**Comment by zekun000** [comment]
> Did it really speed up the benchmark by removing the txn generating thread while executor is doing its work? Does it help if we run the benchmark on a better machine? This limits the scale at which we run the benchmark.

it's ~10% difference, I run it on my devserver. do you offer your better machine to me?

**Comment by msmouse** [comment]
> do you offer your better machine to me?
How about the backup coordinator pod in your testnet? -_,-

**Comment by davidiw** [comment]
Is this outdated? feel free to re-open.

**Pull request closed without merging**